### PR TITLE
mention note numbers for pitch type in readme

### PR DESCRIPTION
**Pull request by alex-wilmer**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,9 +89,11 @@ and the use of `addNoteOn()`/`addNoteOff()` to produce chords.
 
 ### Midi.Track
 
-Note: time and duration are specified in "ticks", and there is a hardcoded
+Time and duration are specified in "ticks", and there is a hardcoded
 value of 128 ticks per beat. This means that a quarter note has a duration of 128.
 
+Pitch can be specified by note name with octave (`a#4`) or by note number (`60`). *60 is middle c*.
```
Comment (dingram) [inline README.md]: I'd change the second part to:

    Middle C is represented as `c4` or `60`.

and start it on the next line.

Comment (alex-wilmer) [inline README.md]: Probably worth mentioning that in many cases middle C is `c3` like for DAWs (ableton / logic etc)

Comment (dingram) [inline README.md]: True, but in this library it's c4 (which I believe is mentioned elsewhere in the README) :-)

**Comment by alex-wilmer** [comment]
updated

File: README.md [hunk 2]
```diff
@@ -89,9 +89,13 @@ and the use of `addNoteOn()`/`addNoteOff()` to produce chords.
 
 ### Midi.Track
 
-Note: time and duration are specified in "ticks", and there is a hardcoded
+Time and duration are specified in "ticks", and there is a hardcoded
 value of 128 ticks per beat. This means that a quarter note has a duration of 128.
 
+Pitch can be specified by note name with octave (`a#4`) or by note number (`60`).
+<br />
```
Comment (dingram) [inline README.md]: No need for the `<br />` here. It's fine to have as obviously part of the same paragraph, I just wanted it to be nicely wrapped here in the plaintext version :-)

Comment (alex-wilmer) [inline README.md]: Oh lol, okay.. one sec

**Comment by alex-wilmer** [comment]
`<br />` removed.

**Comment by dingram** [comment]
Awesome, thanks!

**Pull request merged**